src/main/java/com/hillel/items_exchange/model/User.java

```diff
@@ -48,9 +48,15 @@
             inverseJoinColumns = {@JoinColumn(name = "deal_id", referencedColumnName = "id")})
     private List<Deal> deals;
 
-    @OneToMany(mappedBy = "user", cascade = CascadeType.ALL)
-    private List<Phone> phones;
+    @OneToMany(mappedBy = "user", cascade = CascadeType.ALL, orphanRemoval = true)
+    private Set<Phone> phones;
```

Comment:
Perhaps it is better to use the collection here, in case we will need a List in the future?

Reply:
Actually I had to undo this changes, because I don't use this collection any more in this task. I think it is better not to make changes in User class at all. What do you think about it?